.teamcity/_self/projects/MarTech.kt

```diff
@@ -0,0 +1,136 @@
+package _self.projects
+
+import Settings
+import _self.bashNodeScript
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.perfmon
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.notifications
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.BuildFailureOnMetric
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.failOnMetricChange
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.schedule
+
+object MarTech : Project({
+	id("MarTech")
+	name = "MarTech"
+	description = "Tasks run by MarTech."
+
+	params {
+		param("docker_image", "%docker_image_e2e%")
+	}
+
+	buildType(ToSAcceptanceTracking)
+})
+
+object ToSAcceptanceTracking: BuildType ({
+	name = "ToS Acceptance Tracking"
+	description = "Captures screenshots of locations where Terms of Service are shown."
+
+
+	artifactRules = """
+		screenshots => screenshots
+	""".trimIndent()
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	params {
+		param("env.NODE_CONFIG_ENV", "test")
+		param("env.PLAYWRIGHT_BROWSERS_PATH", "0")
+		param("env.TEAMCITY_VERSION", "2021")
+		param("env.HEADLESS", "false")
+		param("env.LOCALE", "en")
+	}
+
+	steps {
+		bashNodeScript {
+			name = "Prepare environment"
+			scriptContent = """
+				# Install deps
+				yarn workspaces focus wp-e2e-tests @automattic/calypso-e2e
+
+				# Build packages
+				yarn workspace @automattic/calypso-e2e build
+			""".trimIndent()
+			dockerImage = "%docker_image_e2e%"
+		}
+
+		bashNodeScript {
+			name = "Capture screenshots"
+			scriptContent = """
+				# Configure bash shell.
+				shopt -s globstar
+				set -x
+
+				# Enter testing directory.
+				cd test/e2e
+				mkdir temp
+
+				# Decrypt config
+				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%E2E_CONFIG_ENCRYPTION_KEY%"
+
+				# Run suite.
+				xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%E2E_WORKERS% --group=legal
+			"""
+			dockerImage = "%docker_image_e2e%"
+		}
+
+		bashNodeScript {
+			name = "Collect results"
+			scriptContent = """
+				set -x
+
+				mkdir -p screenshots
```

Comment:
One thought is to make this name more specific, just in case anything else tries to use it. Maybe `tos_tc_screenshots`?

Reply:
Good suggestion. I'll make the changes.